modules/create-lokistack-cr-cli.adoc

```diff
@@ -40,11 +41,17 @@ spec:
     mode: openshift-logging
 ----
 <1> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
++
+[IMPORTANT]
+====
+The deployment size cannot be modified after the `LokiStack` CR has been created.
```

Comment:
That is not correct.   "Changing the number in the LokiStack size is not possible". that doesn't means we can not 
change the t-shirt size. we can only using 1x.<size> currently.  But we can update t-shift from 1x.demo to 1x.extra-small.